File: Simple calc.js
// receive operation
var operation = prompt("what operation would you like to use?\n[addition]-[subtraction]-[multiplication]-[division]")
// prompt user for 1st number
var num1 = parseFloat(prompt("what number would you like to " + operation))
// prompt user for 2nd number
var num2 = parseFloat(prompt("and whats the 2nd number you would like to " + operation))
// make sure operation received is valid
if (operation.toLowerCase() === "addition") {
// if addition was selected add and print.
  var answer = num1 + num2
  console.log(answer)
// if operstion subtraction is selected.
} else if (operation.toLowerCase() === "subtraction") {
  var answer = num1 - num2
  console.log(answer)
// if operation multiplication is selected.
} else if (operation.toLowerCase() === "multiplication") {
  var answer = num1 * num2
  console.log(answer)
// if operation division is selected.
} else if (operation.toLowerCase() === "division") {
  var answer = num1 / num2
  console.log(answer)
// print if nothing is true
} else {
  console.log("the operation was invalid.")
}

File: SimpleCalc.py
from time import sleep

#avalible operations to perform.

def add(num1, num2):

  result = num1 + num2

  return print(result)

  

def sub(num1, num2):

  result = num1 - num2

  return print(result)

  

def multiply(num1, num2):

  result = num1 * num2

  return print(result)

  

def divide(num1, num2):

  result = num1 / num2

  return print(result)

while True:

#prompting user for a operation

  userinput = str(input("what would you like to do today sir!\n[add][sub][multiply][divide]\n"))

#promt user for 2 numbers 

  num1 = float(input(f"give me your first number to {userinput}\n"))

  num2 = float(input("now the 2nd please!\n"))

#if user enters "add" this is executed.

  if userinput.lower() == "add":

    add(num1, num2)

    break

#if user enters "sub" this is executed.

  elif userinput.lower() == "sub":

    sub(num1, num2)

    break

#if user enters "multiply" this is executed.

  elif userinput.lower() == "multiply":

    multiply(num1, num2)

    break

#if user enters "divide" this is executed.

  elif userinput.lower() == "divide":

    divide(num1, num2)

    break

#else user will be promoted to re-enter values.

  else:

    print(f"sorry {userinput} wasn't a choice! try again.")

    sleep(2)

 

